### PR TITLE
change for second branch

### PR DESCRIPTION
**Pull request by Malte1983**

### CODE REVIEW

File: neuerordner/neu.md [hunk 1]
```diff
@@ -1,5 +1,2 @@
 # Intro
```
Comment (pixxelschubser) [inline neuerordner/neu.md:1]: Maybe add a blank line like you did before:
```suggestion
# Intro

This is my second change and my second branch
```
But I'm just testing code-line commenting, so there's no need to adjust your code, really ;)

**Comment by StefanBehring** [comment]
Good changes!

**Review by pixxelschubser — APPROVED** [review]

File: neuerordner/neu.md (hunk 1 above)
Comment (Malte1983) [reply neuerordner/neu.md:1]: yeah, that is a good idea

**Review by Malte1983 — COMMENTED** [review]
thanks for help

**Pull request merged**